src/coreclr/jit/emitarm64.cpp

```diff
@@ -9300,6 +9378,7 @@ void emitter::emitIns_R_I_I(instruction ins,
 
     id->idIns(ins);
     id->idInsFmt(fmt);
+    id->idInsOpt(opt);
```

Comment:
Interesting that we weren't setting the `opt` before. Makes sense that we need to. I assume it doesn't cause any problems for the existing instructions.

Reply:
Yeah, the existing instructions don't seem to check `opt` anywhere else -- not even to assert that it's `INS_OPTS_NONE`. That's the default value in `emitIns_R_I_I`, so I imagine always initializing `idInsOpt` is fine; I didn't notice any issues.